desktop/install/mac-install.md

```diff
@@ -42,7 +42,7 @@ Your Mac must meet the following requirements to install Docker Desktop successf
 
   > **Note**
   >
-  > Docker supports Docker Desktop on the most recent versions of macOS. That is, the current release of macOS and the previous two releases. As new major versions of macOS are made generally available, Docker stops supporting the oldest version and supports the newest version of macOS (in addition to the previous two releases). Docker Desktop currently supports macOS Catalina, macOS Big Sur, and macOS Monterey.
+  > Docker supports Docker Desktop on the most recent versions of macOS. That is, the current release of macOS and the previous two releases. As new major versions of macOS are made generally available, Docker stops supporting the oldest version and supports the newest version of macOS (in addition to the previous two releases).
```

Comment:
If we'd rather keep the sentence here, it should read:

> Docker Desktop currently supports macOS Big Sur, macOS Monterey, and macOS Ventura.

(Though stating "macOS" before each version also feels redundant to me 😛)